cfme/tests/cloud/test_targeted_refresh.py

```diff
@@ -0,0 +1,201 @@
+import pytest
+
+from cfme import test_requirements
+
+pytestmark = [
+    test_requirements.ec2,
+    pytest.mark.ignore_stream('5.10', 'upstream')
```

Comment:
but in the tests it's `startsin: 5.9`...

Reply:
I don't actually know what it means...It was in test_manual.py so I copied it here.

Reply:
I believe it will not collect the test against 5.10. If it's not desired, it needs to be removed.